features/issues/components/IssueList/Issue-list.tsx

```diff
@@ -64,14 +64,27 @@ const PageNumber = styled.span`
 export function IssueList() {
   const router = useRouter();
   const page = Number(router.query.page || 1);
+  const statusFilter =
+    typeof router.query.status === "string" ? router.query.status : null;
```

Comment:
Destructuring directly from `router.query` returns possible types of `string | string[] | undefined` since you can have an array of string params. This throws errors further down if I then try and use these filters in the Select component, since they can only accept `string` for certain operations. 

My options were adding a type checks as above, or directly asserting `as string`, but the latter doesn't feel like a good solution. I couldn't figure out how to narrow the types anywhere else either.